src/vehicle.cpp

```diff
@@ -2134,8 +2123,7 @@ std::vector<const vehicle_part *> vehicle::get_parts( const tripoint &pos, const
 
 cata::optional<std::string> vpart_position::get_label() const
 {
-    const vehicle_part &part = vehicle().parts[part_index()];
-    const auto it = vehicle().labels.find( label( part.mount.x, part.mount.y ) );
+    const auto it = vehicle().labels.find( label( mount().x, mount().y ) );
```

Comment:
Not expecting you to fix this for this PR, but do you know why labels are stored in this weird external set instead of being part of the vehicle_part() class?  Tracking labels seems unnecessarily complicated and I can't figure out why they haven't been migrated into vehicle_part.

Reply:
> but do you know why labels are stored in this weird external set instead of being part of the vehicle_part() class?

Ask the inventor. The same applies to `vehicle_part::mount`, which is redundant as well. This is propably because one tile can have many vehicle parts, but only *one* label. But a better solution would probably be a `unordered_map<point, string>`.